resources/serverless/templates/function-dashboard.yaml

```diff
@@ -1084,7 +1084,9 @@ data:
       "refresh": "10s",
       "schemaVersion": 25,
       "style": "dark",
-      "tags": [],
```

Comment:
do we need tags for other dashboards too ?

Reply:
Yes, we already have tags for all the dashboards. This dashboard had a syntax problem. That's why we were not able to see it on grafana and not realized that it had a missing tag.